tests/conftest.py

```diff
@@ -565,6 +567,123 @@ def minio_data_connection(
         yield secret
 
 
+# OCI Registry
```

Comment:
We should add them under async_job conftest. They are not used in any other places.